workflows/prognostic_c48_run/runtime/loop.py

```diff
@@ -69,7 +68,9 @@ def label(self) -> str:
     def __call__(self, time, state) -> Tuple[Tendencies, Diagnostics, State]:
         return {}, {}, {}
 
-    def get_diagnostics(self, state, tendency) -> Tuple[Diagnostics, xr.DataArray]:
+    def get_diagnostics(
+        self, state, tendency, hydrostatic: bool
```

Comment:
New argument here.

this is another sign that formulating our ML in terms of non-conserved variables results in undesirable coupling between the ML and the FV3GFS model physics/configurations.

I know you are looking for the quick fix, but would it really be that hard to switch our ML procedures to use "heating" rather than temperature changes? For N2F, it seems like that would result in a smaller diff than this PR. Not sure about N2O.

Reply:
This isn't a new coupling... even our coarsening procedure is tied to the FV3GFS model physics via the gravity constant used.

I think switching to using heating is interesting, but not sure I agree it's a minimal change since all of our training datasets would need to change.

Reply:
It would be nice if we could grab the value of the relevant constants from the Fortran state within runtime though.

Reply:
> This isn't a new coupling

The `hydrostatic` argument is a new piece of data being passed to the stepper, and is a breaking change to the Stepper interface, as we envision adding new steppers, this would be a step in the wrong direction. I would feel better about it if it were passed to the `__init__` of the Stepper, rather than changing the signature of Stepper.get_diagnostics.